ops/miningManager.go

```diff
@@ -40,11 +42,12 @@ type MiningMgr struct {
 }
 
 // CreateMiningManager creates a new manager that mananges mining and data requests.
-func CreateMiningManager(ctx context.Context, exitCh chan os.Signal, submitter tellorCommon.TransactionSubmitter) (*MiningMgr, error) {
+func CreateMiningManager(ctx context.Context, exitCh chan os.Signal, submitter tellorCommon.TransactionSubmitter, logger log.Logger) (*MiningMgr, error) {
```

Comment:
looks like you missed my chat comments :smile_cat: 
In golang as standard first argument is context and second is logger.

```suggestion
func CreateMiningManager(ctx context.Context, logger log.Logger, exitCh chan os.Signal, submitter tellorCommon.TransactionSubmitter) (*MiningMgr, error) {
```

Reply:
Ops, I actually missed that. I'll adjust it